app/assets/javascripts/discourse/app/components/quote-button.js

```diff
@@ -34,15 +37,30 @@ export default Component.extend({
   visible: false,
   privateCategory: alias("topic.category.read_restricted"),
 
+  _isFastEditable: false,
+  _displayFastEditInput: false,
+  _fastEditInitalSelection: null,
+  _fastEditNewSelection: null,
+  _isSavingFastEdit: false,
```

Comment:
Why are all these fields prefixed with a `_`?

Reply:
private property not supposed to be set from the outside, eg it's intended to show that the following shouldn't be done:

```
{{quote-button _isSavingFastEdit=true}} 
```

It's all virtual and not enforced but a well know patter in js world. In the future we might move to more advanced js class features, but don't think we introduced this in the codebase yet.